src/sql/workbench/contrib/notebook/browser/cellViews/code.css

```diff
@@ -108,3 +108,17 @@ code-component .hide-component-button {
 	background-position: center;
 	background-color: transparent;
 }
+
+code-component .parameter {
+	position: absolute;
```

Comment:
[nit]: for future work in CSS, please alphabetize properties. It's makes it easier to scan quickly.